src/main/twirl/com/lightbend/lagom/docs/getstartedscala.scala.html

```diff
@@ -19,59 +19,37 @@
 
 
 
-                    <p>
-                        The Lagom Giter8 template makes it easy to get started. The template downloads all dependencies and creates the basic project structure. It can take a few seconds to a few minutes to complete.</p>
-
-                        <p>If you have the prerequisites installed and are familiar with sbt,
-                        follow the steps below to create a Hello World project. Otherwise, see
-                        <a href="@context.path/documentation/@context.currentDocsVersion/scala/Installation.html">detailed
-                        instructions</a> for help setting up the prerequisites and getting started. Prerequisites for using Lagom with Scala include:
-
-
-                    <ul>
-                        <li>JDK 8</li>
-                        <li>sbt 0.13.13</li>
-                    </ul></p>
-                    <p>After confirming the prerequisites, open a console or command window and follow these steps:</p>
-
+                  <p>Before starting, make sure you have JDK 8 installed and
+                  configured. See <a class="" href="https://www.lagomframework.com/documentation/1.3.x/scala/Installation.html#JDK" target="_blank">the Lagom documentation</a> if you are not sure whether you have the right version of Java or to find the link for installing it.</p>
+                  The Lightbend Tech Hub Project Starter makes it simple to create your first project. It uses a Lagom Giter8 template to generate a compressed file that contains: 
+                  <ul>
+                    <li>A distribution of sbt, the Scala build tool</li> 
+                    <li>A Lagom Hello World project.</li>
+                  </ul> 
+                  <h2>Download the project from Tech Hub</h2>
+                  <p>Follow these steps to download a project created from the template:</p>
                   <ol>
-                    <li><p>Create a new directory for your project.</p></li>
-
+                    <li><p>Open the <a class="" href="http://developer.lightbend.com/start/?group=lagom&project=lagom-scala-sbt" target="_blank">Tech Hub Project Starter</a></p></li>
+                      
```

Comment:
Do you it'd be necessary to state the link opens a new window?
`... target="_blank">Tech Hub Project Starter</a> (opens in a new window)</p></li>`

I have no preference, I'll leave it up to you.